frontend/src/components/print/print-nuxt/DownloadNuxtPdfButton.vue

```diff
@@ -3,7 +3,7 @@
     <v-btn color="primary" :loading="loading" outlined @click="generatePdf">
       <v-icon>mdi-printer</v-icon>
       <div class="mx-1">
-        {{ $tc('components.print.printNuxt.downloadNuxtPdfListItem.label') }}
+        {{ $tc('components.print.printNuxt.label') }}
```

Comment:
Both the old and new translation key are wrong according to our convention... Conflicts with #3101

Reply:
I haven't reviewed #3101 but I already thought it would conflict with it 😏 

I think we should allow it though to avoid doubling keys. A component should be able to use a translation key from higher up in the tree, shouldn't it?

Reply:
I don't really agree, but we could discuss to change our convention.

So far we have set the convention that way so that it would be easy to determine where a given translation key is used in the application. Recall that it was a compromise we made for lokalise to put all translations inside a single json file (instead of `<i18n>` blocks in the vue components). It might also be possible in the future to implement relative translation keys (e.g. we could then write `$tc('.label')` instead of `$tc('components.print.printNuxt.downloadNuxtPdfButton.label')`).
Also, this is the only way we can be sure that a change in translation does not have unexpected consequences. Imagine if we wanted to change the list items translation to something longer (because the list has space for it), or add a [list-item-subtitle](https://vuetifyjs.com/en/components/lists/#usage) which explains the difference between layouts #1 and #2, that should not unexpectedly change the text of the button. A different component most of the time implies that there is also a different context, which might necessitate different texts (or even different translations for only some languages).

Again, we can gladly discuss whether the convention still makes sense, or whether we want to change or refine it, e.g. do mixin filenames really have to be in the translation key or not. But until we change the convention, I stand by the point that your current change is wrong :)

Reply:
Makes sense, fine for me. Fixed with https://github.com/ecamp/ecamp3/pull/3119/commits/fa96b3975dee0ee71998a1a76c0f6b14f34ac454